cloki.js

```diff
@@ -116,6 +116,7 @@ async function getContentBody (req) {
  */
 async function genericJSONParser (req) {
   try {
+    if (req.routerPath === '/tempo/api/push') { return JSON.parse(await getContentBody(req)) } /* deal with bad spans w/o content-length */
```

Comment:
@akvlad is this an acceptable way to bypass the content size control for broken agents? (clickhouse URL engine)

Reply:
any suggestion here?